cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/spark/SparkUtils.java

```diff
@@ -155,6 +188,158 @@ public static URLClassLoader createSparkFrameworkClassLoader(ClassLoader parentC
     return new URLClassLoader(urls, parentClassLoader);
   }
 
+  /**
+   * Creates a Zip file which contains two files inside, a "yarn-site.xml", which is serialization of the given
+   * {@link YarnConfiguration}, and a "__spark_conf__.properties", which contains the serialization of the given
+   * spark properties
+   *
```

Comment:
Creates a Zip file which contains two files: a "yarn-site.xml", which is a serialization of the given
{@link YarnConfiguration}; and a "__spark_conf__.properties", which contains the serialization of the given
Spark properties.